docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -9,25 +9,23 @@
    "source": [
     "# Introduction\n",
     "\n",
-    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakeFS-spec` for data versioning. We will use weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
+    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakefs-spec` for data versioning. We will use versioned weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
     "\n",
     "We will do the following:\n",
     "\n",
     "* Environment setup\n",
-    "* lakeFS setup\n",
-    "* Data ingestion\n",
-    "    * Transactions\n",
-    "    * PUT a file\n",
+    "* LakeFS setup\n",
+    "* Data ingestion via transactions\n",
     "* Model training\n",
     "* Updating data and retraining a model\n",
-    "* Accessing data versions and Reproducing Experiments\n",
-    "* Using a tag instead of a commit SHA for semantic versioning\n",
+    "* Accessing data versions and reproducing experiments\n",
+    "* Using tags for semantic versioning\n",
     "\n",
     "To execute the code in this tutorial as a Jupyter notebook, download this `.ipynb` file to a convenient location on your machine. You can also clone the whole `lakefs-spec` repository. During the execution of this tutorial, in the same directory, a folder `data` will be created. We will also download a file `.lakectl.yaml` to the current user's home directory.\n",
     "\n",
     "This tutorial assumes that you have installed `lakefs-spec` in a virtual environment, and that you have followed the [quickstart guide](../quickstart.md) to set up a local lakeFS instance.\n",
```

Comment:
Hmm, and what if the user needs to authenticate first and / or leave the email? Then "ensure you can see the list of repositories" is not a great instruction, since you do not know what to do if you're left without guidance.

Is the YAML file being `urlretrieve`'d a problem? We assume a local quickstart instance, for which this should be fine.